server/db/seed.js

```diff
@@ -0,0 +1,13 @@
+import { Db } from ".";
+
+async function setupDb() {
+  await Db.query("CREATE TABLE IF NOT EXISTS shops (shop TEXT)");
+  await Db.query("CREATE UNIQUE INDEX shops_unique_shop_index ON shops(shop)");
+
+  await Db.query("CREATE TABLE IF NOT EXISTS sessions (id TEXT, data JSON)");
```

Comment:
Also, I wonder if we want to have the `Session` object provide a serialization / deserialization option or if JSON parse / stringify is enough.

Reply:
I think the `JSON.parse` is fine in this case. Maybe in a real-world app someone might want to write a serializer but I don't think it's totally necessary here. `JSON.stringify` to save and `JSON.parse` to load get the job done.